app/lib/features/chat/models/room_list_filter_state/room_list_filter_state.dart

```diff
@@ -8,16 +7,17 @@ part 'room_list_filter_state.freezed.dart';
 Future<bool> roomListFilterStateAppliesToRoom(
   RoomListFilterState state,
   Ref ref,
-  Convo convo,
+  String convoId,
```

Comment:
in order to allow for proper mocking I had to make sure we used providers here rather than a convo-object I can't directly mock.

but in general the plan for all model/provider code was to go from a direct convo-interaction object to higher-level providers anyways, soo ...